addons/stock/wizard/stock_picking_return.py

```diff
@@ -141,8 +141,8 @@ def _create_returns(self):
                 move_orig_to_link |= return_line.move_id
                 # link to siblings of original move, if any
                 move_orig_to_link |= return_line.move_id\
-                    .move_dest_ids.filtered(lambda m: m.state not in ('cancel'))\
-                    .move_orig_ids.filtered(lambda m: m.state not in ('cancel'))
+                    .mapped('move_dest_ids.move_orig_ids')\
```

Comment:
@nim-odoo are you sure, you don't need the intermediate filter anymore?

Reply:
I can still keep it, just in case